types.go

```diff
@@ -159,6 +161,22 @@ const (
 	ZoneMDHashAlgSHA512 = 2
 )
 
+// Used in IPSEC https://datatracker.ietf.org/doc/html/rfc4025#section-2.3
+const (
+	IPSECGatewayNone uint8 = 0
```

Comment:
```suggestion
	IPSECGatewayNone uint8 = iota
```

Reply:
These are RFC-defined constants, not locally defined enum values. Is this not the same reason that the RR codes in `types.go` are written out explictly rather than via iota values, despite a sizeable subset of them being continuous values?

Reply:
fair point, but these are just sequentially assigned (unlike the mess that is RR codes). I would just opt for using less characters here, and the constant name should be such that people use that instead of caring of the actual value

Reply:
Fair enough. Already implemented this in the last commit, marking as resolved.